scripts/ci/libraries/_push_pull_remove_images.sh

```diff
@@ -104,37 +104,58 @@ function push_pull_remove_images::pull_image_github_dockerhub() {
     set -e
 }
 
-# Pulls CI image in case caching strategy is "pulled" and the image needs to be pulled
-function push_pull_remove_images::pull_ci_images_if_needed() {
-    if [[ "${DOCKER_CACHE}" == "pulled" ]]; then
-        local python_image_hash
-        python_image_hash=$(docker images -q "${AIRFLOW_CI_PYTHON_IMAGE}" 2> /dev/null || true)
-        if [[ -z "${python_image_hash=}" ]]; then
-            FORCE_PULL_IMAGES="true"
-        fi
-        if [[ "${FORCE_PULL_IMAGES}" == "true" ]]; then
-            echo
-            echo "Force pull base image ${PYTHON_BASE_IMAGE}"
-            echo
-            if [[ -n ${DETECTED_TERMINAL=} ]]; then
-                echo -n "
-Docker pulling ${PYTHON_BASE_IMAGE}.
+# Pulls base python image used to build CI and PROD imaages, depending on the parameters used
```

Comment:
Typo

Reply:
I reworded it a bit including `python -> Python` if that was the typo (the function changed - it is really Python base image that is pulled by the function).

Reply:
It was `imaages`, sorry. Was reviewing on my phone which makes it harder to be verbose. Not important.